onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

```diff
@@ -36,9 +36,11 @@ OrtStatus* ORT_API_CALL RegisterCustomOps(OrtSessionOptions* options, const OrtA
     Ort::CustomOpDomain domain_v2{"v2"};
     Cpu::RegisterOps(domain_v2);
 
+#ifdef USE_CUDA_MINIMAL
     Cuda::RegisterOps(domain);
     Cuda::RegisterOps(domain_v2);
-
+#endif
```

Comment:
If a memcpy would be used as custom op that should work without needing cvcc during compile. Let me know if that would be an accepted change.

Reply:
Not very clear to me why we don't test Cuda::RegsoterOps() in not USE_CUDA_MINIMAL, i.e. normal cuda build now?